docs/odoo/ir.config_parameter.rst

```diff
@@ -0,0 +1,23 @@
+ir.config_parameter
+===================
+
+XML operations
+--------------
+
+Create new setting
+^^^^^^^^^^^^^^^^^^
+Code::
+
+        <record id="myid" model="ir.config_parameter">
+            <field name="key">auth_signup.allow_uninvited</field>
+            <field name="value">True</value>
+        </record>
+
```

Comment:
@ilyasProgrammer add a note, that such **<record** usually used with noupdate="1"